signal/micro/kernels/fft_test.cc

```diff
@@ -220,12 +220,18 @@ TF_LITE_MICRO_TEST(RfftTestSize512Float) {
   float output[kOutputLen * 2];
   const TFLMRegistration* registration =
       tflite::tflm_signal::Register_RFFT_FLOAT();
+// See (b/287518815) for why this is needed.
+#if defined(ARMCM55)
+  float tolerance = 1e-5;
+#else   // defined(ARMCM55)
+  float tolerance = 1e-7;
+#endif  // defined(ARMCM55)
```

Comment:
lmk if you would like to just statically increase tolerance for all tests if you think it can be a pain point in the future.

Tested locally with `make -f tensorflow/lite/micro/tools/make/Makefile -j24 test_kernel_fft_test TARGET=cortex_m_corstone_300 TARGET_ARCH=cortex-m55`

Reply:
Let's increase to e-5 across the board

Reply:
Done.